apps/nowcasting-app/pages/index.tsx

```diff
@@ -19,14 +20,14 @@ export default function Home() {
         <Header view={view} setView={setView} />
         <div
           id="map-container"
-          className={`relative float-right h-full ${currentView(VIEWS.FORECAST) ? "" : "hidden"}`}
+          className={`relative float-right h-full ${currentView(VIEWS.FORECAST)}`}
           style={{ width: "56%" }}
         >
           <PvLatestMap />
         </div>
 
         <SideLayout>
-          <PvRemixChart className={currentView(VIEWS.FORECAST) ? "" : "hidden"} />
+          <PvRemixChart className={`${currentView(VIEWS.FORECAST)}`} />
```

Comment:
If no other characters in this string literal, doesn't need the extra brackets and back ticks I think?

Reply:
typescript is telling me off for taking the back ticks out. that might be why they're there.